General/ExportFormats.md

```diff
@@ -0,0 +1,138 @@
+| Easy to merge two branches | :white_check_mark: | :white_check_mark: | :x: |
+| Takes a reasonable space on file system | :white_check_mark: | :x: | :x: |
+| Works with Iceberg | :white_check_mark: | :white_check_mark: | :x: |
+| Works with Monticello | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+### Supported Pharo versions
+
+| Pharo Version | FileTree metadata full | FileTree metadata less | Tonel |
+| ------------- | ------------- |------------- |------------- |
+3 |  :white_check_mark: | Update Metacello? | :x: |
+4 |  :white_check_mark: |  :white_check_mark: | Update Metacello + Install Tonel |
+5 |  :white_check_mark: |  :white_check_mark: | Update Metacello + Install Tonel |
+6.0 |  :white_check_mark: |  :white_check_mark: | Update Metacello + Install Tonel |
+6.1 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+7 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+8 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+## Tonel 
+
+Tonel ([https://github.com/pharo-vcs/tonel](https://github.com/pharo-vcs/tonel)) is an export format introduced in Pharo 6.1. 
+
+It creates one file by class or extented class and works well on windows (compared to file tree).
+
+### Tonel Pros and Cons
+
+Pros
+- Works well on windows 
+- Not much space lost because of small files
+- Export readable files
+- No problem during merge because of metadata
+
+Cons
+- Work out of the box only since Pharo 6.1
+- Since the format is more recent than the others, some edges cases might cause trouble. (It rarely happens)
+
+### Tonel supported Pharo versions
+
+Tonel works out of the box in Pharo 6.1 and higher. But it is possible to make it work in Pharo 4 to 6.0 by updating Metacello and installing Tonel.
+
+```Smalltalk
+Metacello new
+  baseline: 'Metacello';
+  repository: 'github://Metacello/metacello:master/repository';
+  get.
+Metacello new
+  baseline: 'Metacello';
+  repository: 'github://Metacello/metacello:master/repository';
+  onConflict: [:ex | ex allow];
+  load.
+Metacello new 
+	repository: 'github://pharo-vcs/tonel';
+	baseline: 'Tonel';
+	load.
+```
+
+## FileTree metadata less 
+
+FileTree ([https://github.com/dalehenrich/filetree](https://github.com/dalehenrich/filetree)) is the first export format planned to use Pharo with Git that was integrated. It had a first version with a lot of metadata (see section Filetree metadata full) then a new version with a metadata less format. 
+
+This format exports one file per methods. This can cause trouble with Windows because Windows has a maximum path and file name length of 255 characters. This limit can easily be reached with FileTree. 
+
+Another problem of the FileTree format is that most Pharo's method are short and creating one file by method waste a lot of space because the physical representation of files on a drive have a minimal size (4Ko on NTFS file systems) that is reached by most of Pharo's method. 
```

Comment:
```suggestion
Another problem of the FileTree format is that most Pharo's methods are short and creating one file by method waste a lot of space because the physical representation of files on a drive has a minimal size (4 KiB on NTFS file systems) that is reached by most of Pharo's method. 
```